docs(portal): add warning for auth0 configuration [TDX-2739] (#5005)
Kong/docs.konghq.com: app/konnect/dev-portal/applications/dynamic-client-registration/auth0.md

6. Note the **identifier** value used above, which is also known as the **audience**, as it will be used as the **Client Token Audience** value in {{site.konnect_short_name}}

{:.important}

>    > **Important:** In order for DCR with Auth0 to work within {{site.konnect_short_name}}, the global **OIDC Dynamic Application Registration** setting in Auth0 must be set to `disabled`. If the global **OIDC Dynamic Application Registration** setting is set to `enabled`, the Auth0 Management API will return an error when attempting to create a client.

Reviewer: ```suggestion
   > **Important:** When using DCR with Auth0 in {{site.konnect_short_name}}, the global **OIDC Dynamic Application Registration** setting in Auth0 must be set to `disabled`. If the global **OIDC Dynamic Application Registration** setting is set to `enabled`, the Auth0 Management API will return an error when attempting to create a client.
```
